src/js/lib/monster.ui.js

```diff
@@ -3093,6 +3102,36 @@ define(function(require) {
 		}
 	};
 
+	/**
+	 * Get handlebars templateto render an SVG icon
+	 * @param   {Object} args
+	 * @param   {String} args.id                   Icon ID
+	 * @param   {String} [args.cssClass=svg-icon]  CSS classes to be applied to the SVG tag
+	 * @return  {String}                           SVG icon template
+	 */
+	function getTemplateSvgIcon(args) {
+		if (!_.isPlainObject(args)) {
```

Comment:
`else` is not necessary because the function finishes with the `throw` statement, and also to follow the convention used in the new functions like `randomString` at monster.util.js.